docs/DeveloperGuide.md

```diff
@@ -303,32 +306,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 2a. The list is empty.
+- 2a. The list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+- 3a. The given index is invalid.
+
+  - 3a1. AddressBook shows an error message.
 
-    * 3a1. AddressBook shows an error message.
+    Use case resumes at step 2.
 
-      Use case resumes at step 2.
+**Use case: UC2 Load contact from save file**
 
-*{More to be added}*
+**Actor: `BizBook`**
+
+**Guarantees:** Contact list is initialized with available data.
+
+**MSS**
+
+1.  `BizBook` searches OS directory for save file.
```

Comment:
I think it might be good to define "Operating System (OS)" in just the first instance of using it?

Reply:
We can leave it in the glossary